samples/JwtBearerSample/Startup.cs

```diff
@@ -59,7 +57,7 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     OnAuthenticationFailed = c =>
                     {
-                        c.HandleResponse();
+                        c.Ignore();
```

Comment:
So these need to be named as a pair to make sense.

`context.[Response]Handled()` // Response was sent, terminates the request all the way to UseAuth
vs 
'context.Stop/IgnoreHandler()' // Stop/Ignore just this handler